metals/src/main/resources/db/migration/V3__Jar_symbols.sql

```diff
@@ -0,0 +1,17 @@
+-- Do not allow more than one combination of path and digest
+create table indexed_jar(
+  id int auto_increment,
```

Comment:
what's the expected cardinality for this `id`? If it grows unbounded, maybe it's better to use `bigint`

Reply:
These are unique Jar references (combination of path, size and modification date) over the lifetime of a project on your local drive. I doubt the combinations will ever exceed 2^31.

Reply:
sounds good, let's save space then and leave `int` :)